docs/Ways-of-Working/Guides/Backlog-Refinement-Session-Guidelines.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Guide to Backlog Refinement Sessions
+authors:
+  - Amy Gray
+reviewed:
+reviewer:
+next-review: 2024-06-01
+---
+
+## Backlog Refinement
+
+The purpose of this meeting is for the team to review the prioritised items in the backlog before accepting them. We will make necessary adjustments for clarity and estimate the effort it will take to complete each Backlog Item.
+
+Refinement sessions help to accumulate **a shared, comprehensive understanding of requirements** among the dev team reducing the level of unplanned rework.
+
+As a result of the refining activities, the next chunk of backlog items is ready for delivery meaning that it has the necessary level of transparency for its accurate implementation.
+
+### Desired Outcome
+
+The outcome of the backlog refinement is a list of BLIs that should be clear to all team members and be ready to be implemented in an upcoming Sprint.
+
+Later, at the Sprint planning event, the team agrees a set of BLIs for the Sprint and plans the tasks it will take to complete them.
+
+### Attendees
+
+Ideally, the entire team will attend a Refinement sessions, this includes developers, testers, product managers, and UX when required.
+
+### Prerequisites
+
+The team lead and product manager will groom the backlog prior to refinement, this may include:
+
+- Engaging with key stakeholders to gather requirements
+- Removing BLIs that are no longer relevant
+- Prioritising the BLIs
+- Review BLIs that are ready for refinement, ensuring that there is enough to understand the resources involved to complete it
+- Prior to refinement, a list of BLIs that are due to be refined will be made available for the team. It is recommended that team members review the list prior to the meeting
+
+### Agenda
+
+#### Backlog Item Discussion
+
+Discuss what needs to be done and how it needs to be done to satisfy the acceptance criteria.
+When discussing BLIs the team should:
+
+- Ensure Acceptance Criteria is valid
+- Review dependencies
+- Review order of work
+- Review tasks
```

Comment:
Would tasks be created as a part of a Backlog Refinement session? Or are these created another time?

Reply:
I feel like creation of tasks is something we would do in Backlog Refinement, good point!